compliance/collection/nodeinventorizer/nodeinventory.go

```diff
@@ -21,10 +24,18 @@ type NodeInventoryCollector struct {
 
 // Scan scans the current node and returns the results as storage.NodeInventory object
 func (n *NodeInventoryCollector) Scan(nodeName string) (*storage.NodeInventory, error) {
-	log.Info("Started node inventory")
+	log.Debug("Start collecting node inventory")
+	metrics.ObserveScansTotal(nodeName)
+	startTime := time.Now()
+
 	// uncertifiedRHEL is set to false, as scans are only supported on RHCOS for now,
 	// which only exists in certified versions
 	componentsHost, err := nodes.Analyze(nodeName, "/host/", nodes.AnalyzeOpts{UncertifiedRHEL: false, IsRHCOSRequired: true})
+
+	scanDuration := time.Since(startTime)
+	metrics.ObserveScanDuration(scanDuration, nodeName, err)
+	log.Debug("Finished collecting node inventory")
```

Comment:
Not sure we need this at the debug level. It looks more like println-debugging. I would vote to remove this.

Reply:
This was added to get a rough estimate of scan times. Now that we have the metrics, we might not need this anymore.
But if we remove this, I would also vote to remove the debug message noting the beginning of a Node Scan as well.

Reply:
I see. Then I would suggest to rephrase this to sth like: `log.Debugf("Collecting node inventory took %ss", scanDuation.Seconds().String())`

Reply:
That's a great idea, let me rework this